recipes/r-smoothr/meta.yaml

```
@@ -0,0 +1,91 @@
+{% set version = '1.0.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-smoothr
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/smoothr_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/smoothr/smoothr_{{ version }}.tar.gz
+  sha256: 580b150009934475173a6fc917020bc423934103021a8d3e6b6620d2b93fc5e9
+
+build:
+  skip: true  # [win]
```

Its a noarch package as it does not contain compiled code... the windows failures are to be ignored, as the package will be build on linux only after the feedstock conversion.
```suggestion
  noarch: generic
```